ocrd/bashlib/src/parse_argv.bash

```diff
@@ -78,7 +78,7 @@ ocrd__parse_argv () {
     if [[ -n "${ocrd__argv[page_id]:-}" ]]; then
         _valopts+=( --page-id "${ocrd__argv[page_id]}" )
     fi
-    _valopts+=( "${OCRD_TOOL_NAME#ocrd-} -I ${ocrd__argv[input_file_grp]} -O ${ocrd__argv[output_file_grp]}" )
+    _valopts+=( "${OCRD_TOOL_NAME#ocrd-} -I ${ocrd__argv[input_file_grp]} -O ${ocrd__argv[output_file_grp]} ${__parameters[*]} ${__parameter_overrides[*]}" )
```

Comment:
I am wondering whether I should not quote the parameter variables here, though: Parameter values might themselves contain white-space, which would break in the `task_sequence.parse` because the `-p` or `-P` needs exactly two arguments.

Reply:
But quoting would clog all the `-p` / `-P` arguments into one. Perhaps the only adequate solution is to use 
```suggestion
    _valopts+=( "${OCRD_TOOL_NAME#ocrd-} -I ${ocrd__argv[input_file_grp]} -O ${ocrd__argv[output_file_grp]} ${__parameters[*]@Q} ${__parameter_overrides[*]@Q}" )
```

Reply:
```
Q
The expansion is a string that is the value of parameter quoted in a format that can be reused as input.
```

https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html

Reply:
spoiler: it does work.